meshack-mbuvi/ride-my-way-api
# Ch refactor tests 158942046

#### What does this pr do?
It groups related tests into a single file.

 #### Description of Task to be completed?
- Have tests for related endpoints in a single file

#### How should this be manually tested?
- clone this repo
- cd ride-my-way
- set and activate your virtual environment
- git checkout ch-refactor-tests-158942046
- Open test folder and confirm tests for related endpoints are grouped in a single file
- Run nosetests -v to confirm all tests run successfully
